src/main/kotlin/no/nav/cv/event/cv/CvEndretProcessor.kt

```diff
@@ -25,8 +25,8 @@ class CvEndretProcessor (
     ) {
         val cv = CvDto(record.value())
 
-        if(!cv.slettetCv()) {
-            log.info("Kafka CV msg for ${cv.aktorId()} is NOT a deletion msg")
+        if(!cv.slettetCv() && cv.endretAvBruker()) {
```

Comment:
Dette vil jo være false for alle gamle avro meldinger, og frem til CV api endringene har blitt satt i drift. Jeg tenker at det er helt greit, så lenge vi ikke setter dette i drift før CV api begynner å sende feltet.